### PR TITLE
RC65: Fix addressing the avatar mesh visibility switch bug

### PR DESCRIPTION
**Pull request by samcake**
This PR addresses this bug [13164](https://highfidelity.manuscript.com/f/cases/13164/Avatar-is-still-visible-in-away-mode)

The problem is that MyAvatar::setEnableMeshVisible which is used to toggle the mesh on and off is not telling the Meta RenderItem oto switch visibility accordingly.
It only tells the model's RenderItems representing the parts of the shape.
It s not enough because now , avatar's model RenderItem are all "GroupCulled" by the meta item.
and so since the Avatar Meta item is always visible, the subs are alwyas visible.

I temporarily fixed in this branch by having the MetaREnderItem adjusting it's key's visibility on the spot by going query the visisbility of the avatar model.
THis is not ideal but it works for now.
WE will need a better fix for RC66.

## TEST PLAN ##
In RC65 branch, my Avatar is always visible. It should  be hidden when i press the the "Esc" key to be away or when using the advanced developer menu item : Developer/Avatar/DrawMesh

IN THis PR, this is working correctly, with the Avatar model hidden when pressing escape or when using the checkbox.

### CODE REVIEW

File: libraries/avatars-renderer/src/avatars-renderer/Avatar.cpp [hunk 1]
```diff
@@ -52,7 +52,13 @@ const glm::vec3 HAND_TO_PALM_OFFSET(0.0f, 0.12f, 0.08f);
 
 namespace render {
     template <> const ItemKey payloadGetKey(const AvatarSharedPointer& avatar) {
-        return ItemKey::Builder::opaqueShape().withTypeMeta().withTagBits(ItemKey::TAG_BITS_0 | ItemKey::TAG_BITS_1).withMetaCullGroup();
```
Comment (SamGondelman) [inline libraries/avatars-renderer/src/avatars-renderer/Avatar.cpp]: Should we keep withMetaCullGroup?

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/8036/app_12646_PR.apk

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/8036/HighFidelity-Beta-PR12646-8106bc36d18da2b30a6eaa76ab877399aefeaba8.exe

File: libraries/avatars-renderer/src/avatars-renderer/Avatar.cpp (hunk 1 above)
Comment (samcake) [reply libraries/avatars-renderer/src/avatars-renderer/Avatar.cpp]: good catch!

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/8036/HighFidelity-Beta-PR12646-8106bc36d18da2b30a6eaa76ab877399aefeaba8.dmg

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/8039/app_12646_PR.apk

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/8039/HighFidelity-Beta-PR12646-653bacf27f4c2dc630d6f7889d24320f4d334705.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/8039/HighFidelity-Beta-PR12646-653bacf27f4c2dc630d6f7889d24320f4d334705.dmg

**Comment by samcake** [comment]
SInce this is postponed to rc66 i made a different pr for the master branch
closing this one

**Pull request closed without merging**